pkg/eve/installer/grub_installer.cfg

```diff
@@ -34,28 +34,32 @@ function installer_submenus {
    menuentry 'set the target persist disk' {
       set_global dom0_cmdline "$dom0_cmdline eve_persist_disk=$eve_persist_disk"
    }
-}
 
-function set_global {
```

Comment:
`set_global` function is removed here but I don't see where else it's defined... why it was removed?

Reply:
No longer need it here. The main `grub.cfg` (which is `pkg/grub/rootfs.cfg`) comes first, and it has a line to include `grub_include.cfg` (which is injected from this `grub_installer.cfg`). So common definitions already are there.

It also makes it significantly easier to understand the flow of grub:

1. Main `grub.cfg` is first
2. Main `grub.cfg` includes other file, if found
3. We can expand it to more of those, if and when we want

The only exceptions are when access to main `grub.cfg` needs to be created, e.g. booting off of an El Torito `boot.img`, or from iPXE, in which case we might have a small `grub.cfg` that knows how to find the main one and launch it, but it doesn't set anything.

Reply:
ok, got it, thanks...